### DISCUSSION TITLE
Throwing a window to a floating space fullscreens it

### DISCUSSION THREAD

**Discussion by gkshenaut** [ianyh/Amethyst#1473] [Q&A] [answered]
I'm not sure whether this is an annoyance or a feature. For me, it's the former. When you throw a window to another space, if the destination space is floating, the window becomes fullscreen. Usually, I do not want this. I would prefer the window remain the same size unless one or both of its dimensions are too large, in which case the overlarge dimension(s) should be reduced to fit. I've looked for an option to set this mode as default, but didn't find any such option.

Personally, I consider this to be a bug, in the sense that it is not optimal for my workflow (however, it is consistent and predictable). Ideally, there would be an option to set or disable resizing on throw to a floating space.

IIRC, this is a relatively new behavior for Amethyst, but I could be mistaken.

Thanks for reading.

**Comment by ianyh** [comment, marked as answer]
Yup! It's a bug. See #1300.

**Comment by gkshenaut** [comment]
Thanks
